index.d.ts

```diff
@@ -0,0 +1,95 @@
+export interface Options {
+  /**
+   * Window or WebView to add the context menu to.
+   * When not specified, the context menu will be added to all existing and new windows.
+   */
+  window?: Electron.BrowserWindow | Electron.WebviewTag;
+
+  /**
+   * Should return an array of [menu items](https://electronjs.org/docs/api/menu-item) to be prepended to the context menu.
+   */
+  prepend?: (params: Electron.ContextMenuParams, browserWindow: Electron.BrowserWindow | Electron.WebviewTag) => Electron.MenuItem[];
+
+  /**
+   * Should return an array of [menu items](https://electronjs.org/docs/api/menu-item) to be appended to the context menu.
+   */
+  append?: (param: Electron.ContextMenuParams, browserWindow: Electron.BrowserWindow | Electron.WebviewTag) => Electron.MenuItem[];
+
+  /**
+   * Show the `Copy Image Address` menu item when right-clicking on an image.
+   *
+   * @default false
+   */
+  showCopyImageAddress?: boolean;
+
+  /**
+   * Show the `Save Image As…` menu item when right-clicking on an image.
+   *
+   * @default false
+   */
+  showSaveImageAs?: boolean;
+
+  /**
+   * Force enable or disable the `Inspect Element` menu item.
+   *
+   * Default: [Only in development](https://github.com/sindresorhus/electron-is-dev)
+   */
+  showInspectElement?: boolean;
+
+  /**
+   * Overwrite labels for the default menu items. Useful for i18n.
+   *
+   * @default {}
+   *
+   * @example
+   *
+   * labels: {
+   * 	cut: 'Configured Cut',
+   * 	copy: 'Configured Copy',
+   * 	paste: 'Configured Paste',
+   * 	save: 'Configured Save Image',
+   * 	saveImageAs: 'Configured Save Image As…'
+   * 	copyLink: 'Configured Copy Link',
+   * 	copyImageAddress: 'Configured Copy Image Address',
+   * 	inspect: 'Configured Inspect'
+   * }
+   */
+  labels: object;
```

Comment:
Don't use generic types like `object` or `Function`. Use specific type-signatures like `{[key: string]: number}` or `(input: string) => boolean;`.

Reply:
Then you can just drop the example.